test/integration/targets/openssl_csr/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+- import_role:
+    name: setup_openssl
```

Comment:
Please use meta dependencies instead of import_role here and in the other tests.

Example: https://github.com/ansible/ansible/blob/devel/test/integration/targets/mysql_db/meta/main.yml

This will permit ansible-test to properly handle the role dependencies when analyzing changes from PRs.